ChangeLog.md

```diff
@@ -0,0 +1,29 @@
+# [if me](https://www.if-me.org/) ChangeLog
+
+## Major bug fixes and noteworthy changes
+
+2017-09-27
+
+file added:
+
+* ChangeLog.md (#651)
+
+--
+
+2017-09-25
+
+feature added:
+
+* Secret Share (#455)
+  * ability to share secret links to Moments posts so that non if me users can
```

Comment:
Replace with just:

```
* Ability to share secret links to Moments posts so that unregistered users can view them
* Worked on by @jnamster, @SophieRoseMcDonald, @TimMoore, @HashNotAdam, and @simonhildebrandt
```